go/host/host.go

```diff
@@ -302,11 +306,17 @@ func (a *Node) startProcessing() {
 		select {
 		case b := <-a.blockRPCCh:
 			roundInterrupt = triggerInterrupt(roundInterrupt)
-			a.processBlocks([]common.EncodedBlock{b.p, b.b}, roundInterrupt)
+			err := a.processBlocks([]common.EncodedBlock{b.p, b.b}, roundInterrupt)
+			if err != nil {
+				common.LogWithID(a.shortID, "Could not process block received via RPC. Cause: %v", err)
```

Comment:
Tiny bit annoying to add `WarnWithID()` to logging.go but feels like maybe these should be warn level messages?

Getting dodgy blocks might not be fatal but feels like something operators would want flagged up (since atm I guess it likely will be fatal and this could help debug).